.github/CONTRIBUTING.md

```diff
@@ -43,6 +41,17 @@ After writing content, [submit your work](https://github.com/magento/merchdocs/w
 
 **Tip!** If you are not sure where to start contributing, search for issues with the `help wanted` and `good first issue` labels. These are researched and ready to work!
 
+## User Guide versions
+
+Our repository currently publishes two versions of the Magento User Guide: 2.3.x and 2.4.x (current).
+
+If you want to contribute new or updated content, make sure that you create your working branch from the correct production branch:
+
+- 2.4.x - `master`
+- 2.3.x - `2.3-production`
+
+Use the same branch as the base when you submit the PR. If you indicate in your PR description that the content applies across versions, one of our staff members will address any back port or forward port that is needed.
```

Comment:
We should port any changes only to older versions 2.4 -> 2.3. No porting like 2.3 -> 2.4.

Reply:
I am in disagreement with this, as I have used this type of version management before, and forward is cleaner than backward. Some things will accommodate a cherry pick, while others will need further examination and must be done specifically for each release.

Reply:
cc @jeff-matthews

Reply:
Manual porting (of any kind) in the merchdocs repo will require skilled oversight by @leslietilling.

Reply:
Yes, by `one of our staff members` I basically mean me ;-)

Reply:
haha! i figured as much, but thought it prudent to be specific 😉